lib/core/src/lib/form/components/widgets/core/form-field-validator.spec.ts

```diff
@@ -708,9 +708,9 @@ describe('FormFieldValidator', () => {
 
         it('should take into account that max value is in UTC and NOT fail validating value checking the time', () => {
             const maxValueFromActivitiInput = '31-3-2018 12:00 AM';
-            const maxValueSavedInForm = moment(maxValueFromActivitiInput, 'DD-M-YYYY hh:mm A').utc().format();
+            const maxValueSavedInForm = DateFnsUtils.parseDate(maxValueFromActivitiInput, 'DD-M-YYYY hh:mm A').toISOString();
 
-            const localValidValue = '2018-3-30 11:59 PM';
```

Comment:
you are again changing the test data coming  from backend, do not touch this value